### ISSUE TITLE
Request for training guidance of Latent Consistency Fine-tuning

### ISSUE THREAD

**Issue by Luciennnnnnn** [luosiallen/latent-consistency-model#31]
Hi, very exciting work! I'm trying to reproduce your results of LCF on Pokemon dataset. However, I can't reach a convergence for training. I use MSE loss for training, the training loss is always around 0.005 and does not show clear decent trends.

Below is a brief list of my training configurations:
Batch size: 72
EMA rate: 0.999943
Learning Rate: 8e-6
Epoch: 800 (approximately 10k steps)

For the skipping step, I keep same as your pretrained model (it is 20 considering Stable Diffusion, and 1 considering your LCM).

Do you have any ideas regrading this failure? And also could you please provide training code or help me at reproducing your results? Any help would be appreciated!

If it's possible help me at reproducing the results, you may send your vx to my email: 1224084650@qq.com. Thanks!

**Comment by dragon-cao** [comment]
I encounter the same issue. Have you solved this?

**Comment by dragon-cao** [comment]
Hello! Could you share me your LCF code? Thank you!!!
